src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -44,51 +50,41 @@ public void setLeftServoAngle(double angle){
         this.leftServo.setAngle(angle);
     }
     public void setRightServoAngle(double angle){
-        this.rightServo.setAngle(angle);
+        // this.rightServo.setAngle(angle);
+        this.rightServo.setPosition(angle);
     }
 
     /**
-     * sets the speed of both motors to the IntakeSpeed defined in the {@link Constants} file
+     * Right motor - Positive is up, Negative is up
+     * Left motor - Positive is up, Negative is down
      */
-    public void raise(){
-        climberLeft.set(-Constants.CLIMBER_RAISING_SPEED);
-        climberRight.set(-Constants.CLIMBER_RAISING_SPEED);
+    public void setSpeed(double spd) {
+        climberRight.set(-spd);
+        climberLeft.set(spd);
     }
-    public void lower(){
-        climberLeft.set(isLeftLimit() ? 0 : Constants.CLIMBER_RAISING_SPEED);
-        climberRight.set(isRightLimit() ? 0: Constants.CLIMBER_RAISING_SPEED);
-    }
-    public void balanceRight(double speed) {
-        climberLeft.set(speed);
-        climberRight.set(0);
-    }
-    public void balanceLeft(double speed) {
-        climberLeft.set(0);
-        climberRight.set(speed);
-    }
-    public void stop(){
-        climberLeft.set(0);
-        climberRight.set(0);
+
+    public void resetEncoders() {
+        this.climberLeft.getEncoder().setPosition(0);
+        this.climberRight.getEncoder().setPosition(0);
+    }    
+
+    public void engageRatchet() {
```

Comment:
Is this being used?

Reply:
Maybe using these methods is better than setting the angles externally through the command?

Reply:
Agreed, done